dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -144,9 +144,16 @@ void NavigationViewItem::UpdateRepeaterItemsSource()
             }
             return MenuItems().as<winrt::IInspectable>();
         }();
+        m_itemsSourceViewChanged.revoke();
         repeater.ItemsSource(itemsSource);
+        m_itemsSourceViewChanged = repeater.ItemsSourceView().CollectionChanged(winrt::auto_revoke, { this, &NavigationViewItem::OnItemsSourceViewChanged });
     }
 }
+
+void NavigationViewItem::OnItemsSourceViewChanged(const winrt::IInspectable& sender, const winrt::NotifyCollectionChangedEventArgs& args)
```

Comment:
nit: You could comment out the parameters *sender* and *args* here and in NavigationViewItem.h as they are not used inside your event handler. I.e.:
```cpp
void NavigationViewItem::OnItemsSourceViewChanged(const winrt::IInspectable& /*sender*/, const winrt::NotifyCollectionChangedEventArgs& /*args*/)
```

Reply:
I'd actually recommend just deleting the name entirely

Reply:
I opted to comment out the unused args as that is the style used in the NavigationViewItem code. Just had a chance to push out a commit as I have been busy with other projects.